### ISSUE TITLE
Why are the copyright to your project owned by google?

### ISSUE THREAD

**Issue by MWGuy** [illuspas/Node-Media-Server#364]
```
The MIT License

Copyright (c) 2010-2018 Google, Inc. http://angularjs.org
```

I think it's worth changing the license file.

**Comment by Laphatize** [comment]
AngularJS was made by Google, I thought?

**Comment by hthetiot** [comment]
Fixed by Fixed by c4a6abb6816606ff9356498fd10607345ac7aa33 can be closed
